pkg/test/cluster/app.go

```diff
@@ -170,6 +170,11 @@ func (a *app) Call(url string, count int, headers http.Header) (test.AppCallResu
 	return out, nil
 }
 
+// CallFortio implements the test.DeployedApp interface
+func (a *app) CallFortio(cmd string) (test.AppCallFortioResponse, error) {
```

Comment:
I think we should keep this a separate object altogether for now.  Let's make a `FortioApp` (or something like that)

Reply:
+1.